crates/iroha/tests/integration/asset.rs

```diff
@@ -294,8 +231,8 @@ fn client_add_asset_with_name_length_more_than_limit_should_not_commit_transacti
 #[allow(clippy::expect_fun_call)]
 #[test]
 fn find_rate_and_make_exchange_isi_should_succeed() {
-    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(10_675).start_with_runtime();
-    wait_for_genesis_committed(&[test_client.clone()], 0);
+    let (network, _rt) = NetworkBuilder::new().start_blocking().unwrap();
+    let test_client = network.client();
```

Comment:
I suppose a network with 1 peer is created by default? I think that's ok. No need to test consensus every time

However, I hope that `network.client()` returns a random client when a multi-node network is started

Reply:
Made `network.client()` and `network.peer()` both return random ones.